### PR TITLE
adding authentication

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added:login
added:signup
demo:https://objective-jackson-0fc355.netlify.app/authentication/signup/signup

### CODE REVIEW

File: Authentication/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  .form-container{
+      height: 100vh;
+  }
+
+  .label-text{
+      font-size:2rem;
+      margin: 4px 4px;
+  }
+
+.input{
```
Comment (tushargoyal812) [inline Authentication/Login/login.css:33]: suggestion:add space before { .

**Review by tushargoyal812 — COMMENTED** [review]
minor changes only,looks good to me.

File: Authentication/Signup/signup.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  .hover-underline{
+      margin: 8px;
+  }
+
+  .hover-underline:hover{
+    
+      cursor: pointer;
+      text-decoration: underline;
+
```
Comment (muskan-gupta-0803) [inline Authentication/Signup/signup.css:55]: remove unnecessary blank space

File: Authentication/Signup/signup.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  .label-text{
+      font-size:2rem;
+      margin: 4px 4px;
+  }
+
+.input{
+      height: 5rem;
+      width: 60rem;
+      font-size: 2rem;
```
Comment (mohitdhatrak) [inline Authentication/Signup/signup.css:36]: use color white for text in login and signup buttons, there is a contrast issue

**Review by muskan-gupta-0803 — APPROVED** [review]
Looks good to merge. Nicely done.

Just a suggestion, if you remove border from navbar, it might look better. Otherwise its fine and up to you.

**Review by mohitdhatrak — COMMENTED** [review]
Seems good, just the contrast issue suggestion

File: Authentication/Signup/signup.css (hunk 3 above)
Comment (sanjay-xdr) [reply Authentication/Signup/signup.css:36]: oh yes i see

File: Authentication/Signup/signup.css (hunk 2 above)
Comment (sanjay-xdr) [reply Authentication/Signup/signup.css:55]: sure

**Review by HrishikeshMahalle — APPROVED** [review]
Make sure to implement above noted changes. Good Job!

**Pull request merged**